src/service/alert.service.ts

```diff
@@ -121,7 +121,7 @@ export class AlertService {
     }
 
     closeAlert(id: number, extAlerts?: Alert[]): any {
-        let thisAlerts: Alert[] = extAlerts ? extAlerts : this.alerts;
+        let thisAlerts: Alert[] = (extAlerts && extAlerts.length > 0) ? extAlerts : this.alerts;
```

Comment:
The extAlerts is used if you want to provide your own alert collector array so that if you want to close alerts globally like when using toast style you could do so. But I think its not properly documented. I'll try to add docs for that later